src/ast/dml.rs

```diff
@@ -45,6 +45,7 @@ pub struct CreateIndex {
     pub if_not_exists: bool,
     pub include: Vec<Ident>,
     pub nulls_distinct: Option<bool>,
+    pub with: Vec<Expr>,
```

Comment:
It'd be great if can add a link to the docs here.

Reply:
Added.